### PR TITLE
gip-0064 draft

### PR DESCRIPTION
**Pull request by azf20**
Proposes simplifying enablement of new Graph Node versions on The Graph Network

### CODE REVIEW

File: gips/0064-graph-node-version-updates.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+Currently, as specified in GIP-0008, the Graph Council ratifies each Graph Node version via a GGP.
+
+# High-Level Description
+
+The Graph Council should delegate responsibility for specification of the protocol-supported version of Graph Node to the Graph Node core development team. The Graph Council can revoke this delegation of responsibility at any time.
+
+The Graph Node team should provide the Council and community with updates on new versions, along with any salient information regarding functionality or testing.
+
+The Graph Council will still be responsible for enablement of indexing rewards in the feature support matrix, via GGP.
```
Comment (PedroMD) [inline gips/0064-graph-node-version-updates.md:29]: @azf20, do we want to be this specific, mentioning only "indexing rewards" and not other potentially critical things like arbitration support on specific subgraph features?

File: gips/0064-graph-node-version-updates.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+The Graph Council should delegate responsibility for specification of the protocol-supported version of Graph Node to the Graph Node development team.
+
+# Motivation
+
+This GIP seeks to simplify the release process, and reduce bureaucracy, in order to reduce the time it takes for new features and fixes to be supported on the network by indexers.
+
+# Prior Art
+
+Currently, as specified in GIP-0008, the Graph Council ratifies each Graph Node version via a GGP.
```
Comment (PedroMD) [inline gips/0064-graph-node-version-updates.md:21]: a reference to the actual ratified process may be appreciated https://snapshot.org/#/council.graphprotocol.eth/proposal/0xbdd884654a393620a7e8665b4289201b7542c3ee62becfad133e951b0c408444

File: gips/0064-graph-node-version-updates.md (hunk 1 above)
Comment (azf20) [reply gips/0064-graph-node-version-updates.md]: Makes sense, updated

File: gips/0064-graph-node-version-updates.md (hunk 2 above)
Comment (azf20) [reply gips/0064-graph-node-version-updates.md]: Good idea

**Comment by PedroMD** [comment]
@azf20, was this ever discussed with the Council?

**Comment by azf20** [comment]
It was - there was an informal vote in favour, should we follow up with a snapshot vote?

**Comment by PedroMD** [comment]
I don't think we need a formal vote for this if it's been already verbally voted in favor and given also it's mostly an operational change... We can merge this and then update GIP-0008 to reflect the change in the process. Wdyt?

**Comment by pcarranzav** [comment]
I think a GGP will be useful for the Council to publicly show the approval. This is an operational change, yes, but it affects arbitration so it should have an official vote. But I agree modifying GIP-0008 to incorporate this would be good.

**Comment by pcarranzav** [comment]
GGP created here: https://snapshot.org/#/council.graphprotocol.eth/proposal/0x71bb6aafe1ef7e75b5ae5e8ea9c1b29a04d375d09e18194b6c8cd41ac4826873

**Review by pcarranzav — APPROVED** [review]

**Pull request merged**